cosmosdb/common/ps-account-key-get.ps1

```diff
@@ -0,0 +1,7 @@
+# List keys for an Azure Cosmos Account
```

Comment:
Can you merge this script with get connection strings script?

Reply:
done